index.html

```diff
@@ -419,7 +430,42 @@ <h2>ts-help-text</h2>
+					takimata gubergren elitr takimata. Et et amet sed elitr dolore sit, elitr sit est amet accusam dolor ea sed.
+					Aliquyam lorem sit eos nonumy dolores, sed elitr elitr amet sed eos at diam accusam ipsum, sed lorem justo sed
+					accusam. Lorem kasd sed sit ea magna. Amet amet sed ut sed, eirmod ea takimata vero sanctus. Tempor sed sed
+					sit clita sed. Elitr labore et justo invidunt, et ipsum accusam gubergren et diam consetetur justo nonumy, et
+					stet consetetur magna justo est duo et sea. Dolore ea at eos et dolor sit magna, eirmod tempor eirmod vero
+					vero. Erat aliquyam eirmod ipsum est et clita. Takimata lorem justo gubergren magna et et dolor diam,
+					consetetur dolor at sea vero, et consetetur dolores nonumy dolor sed et sed, rebum at ipsum sit ipsum voluptua
+					amet. Dolore sadipscing dolores amet sit sadipscing et duo, amet stet no vero ut. Lorem dolore elitr stet amet
+					consetetur et magna, consetetur magna erat eos ea dolores et gubergren sanctus duo. Diam ipsum et consetetur
+					dolore sed ipsum diam est. Labore sed ea et magna sed nonumy, magna gubergren et amet diam sed sanctus sed.
+					Takimata dolor kasd sed lorem sit ut accusam tempor, lorem amet invidunt lorem dolor elitr ipsum. Est eos
+					dolore dolores ea voluptua takimata aliquyam dolore justo. Erat sed dolor sit amet elitr est eirmod.
+					Sadipscing aliquyam lorem est elitr elitr eos accusam gubergren. Et lorem amet sadipscing takimata diam
+					tempor. Dolore takimata tempor amet lorem. Tempor sed dolor magna eirmod labore. Dolore stet et ipsum aliquyam
+					erat dolor lorem. Amet sed duo voluptua lorem vero amet gubergren ipsum dolor, accusam est sanctus lorem
+					consetetur magna. Duo et ipsum no gubergren sadipscing justo et. Vero accusam dolor voluptua rebum et tempor
+					ipsum, dolor gubergren vero invidunt sit eos accusam consetetur magna amet, et diam tempor et et eirmod kasd,
+					clita lorem lorem stet consetetur rebum dolor diam amet, magna clita no et no clita rebum sed takimata sea,
+					duo lorem kasd sanctus no sanctus takimata ut et justo. Sea sit est amet rebum et at at lorem et, sed takimata
+					erat sed elitr, et gubergren justo accusam justo. Amet at takimata stet dolores lorem. Elitr rebum ea magna
+					sadipscing dolor. Magna eirmod gubergren dolor amet sit stet dolor erat sed. Nonumy sea et at dolor tempor
+					tempor. Dolor dolore lorem eirmod lorem aliquyam ipsum sadipscing dolor vero. Et ea sit ut amet. Sit amet et
+					sadipscing sit magna sit, kasd invidunt sadipscing dolores et no lorem. Justo et rebum eirmod rebum lorem,
+					diam takimata nonumy magna consetetur et, lorem consetetur.
+				</ts-panel>
+				<ts-button-group slot="footer">
+					<ts-button type="primary" onclick="toggleCover('my-cover')">Toggle cover</ts-button>
+					<ts-button onclick="toggleAside('my-aside')">Toggle aside</ts-button>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add a 'onKeyPress' attribute to this <ts-button> tag. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.ts.sv/coding_rules#rule_key=Web%3AMouseEventWithoutKeyboardEquivalentCheck)